android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -727,7 +729,15 @@ public void handleJsonEvent(final JsonReadEvent jsonReadEvent) {
 
                     OpenEventApp.postEventOnUIThread(new MicrolocationDownloadEvent(true));
                     break;
-                } default:
+                }case ConstantStrings.SESSION_TYPES: {
+                    List<SessionType> sessionTypes = objectMapper.readValue(json, new TypeReference<List<SessionType>>() {});
```

Comment:
@arp95 it is Model class. If json file is event then type should be `Event.class`. Same at other places